README.md

```diff
@@ -23,6 +23,24 @@ def deps do
 end
 ```
 
+You'll also need to add `:artifact` to your list of applications, again in `mix.exs`, something like:
```

Comment:
I don't think we need to say "again in `mix.exs` something like:".  How about:

"Next, add `:artifact` to your list of applications:"